core-services/src/main/java/org/zalando/nakadi/service/publishing/AvroEventProcessor.java

```diff
@@ -0,0 +1,46 @@
+package org.zalando.nakadi.service.publishing;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.zalando.nakadi.domain.NakadiRecord;
+import org.zalando.nakadi.domain.NakadiRecordMetadata;
+
+import java.util.List;
+
+@Component
+@Qualifier("avro-publisher")
+public class AvroEventProcessor extends EventsProcessor<NakadiRecord> {
```

Comment:
Though I like `EventProcessor` more, the base class is still called `Event*s*Processor`...